tests/datasets/test_cloud_cover.py

```diff
@@ -0,0 +1,118 @@
+import glob
+import os
+import shutil
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pytest
+import torch
+import torch.nn as nn
+from _pytest.monkeypatch import MonkeyPatch
+
+from torchgeo.datasets import CloudCoverDetection
+
+# TODO use with pytest.fixture params
+# from requests import request
+# from _pytest.fixtures import SubRequest
+# from torch.utils.data import ConcatDataset
+
+
+class Dataset:
+    def download(self, output_dir: str, **kwargs: str) -> None:
+        glob_path = os.path.join(
+            "tests", "data", "ref_cloud_cover_detection_challenge_v1", "*.tar.gz"
+        )
+        for tarball in glob.iglob(glob_path):
+            # print(os.path.exists(tarball))
+            shutil.copy(tarball, output_dir)
+
+
+def fetch(dataset_id: str, **kwargs: str) -> Dataset:
+    return Dataset()
+
+
+class TestCloudCoverDetection:
+    # @pytest.fixture(params=(["train","test"]))
+    @pytest.fixture
+    def dataset(
+        self,
+        monkeypatch: MonkeyPatch,
+        tmp_path: Path,
+        # request: SubRequest
+    ) -> CloudCoverDetection:
+        radiant_mlhub = pytest.importorskip("radiant_mlhub", minversion="0.2.1")
+        monkeypatch.setattr(radiant_mlhub.Dataset, "fetch", fetch)
+
+        test_image_meta = {
+            "filename": "ref_cloud_cover_detection_challenge_v1_test_source.tar.gz",
+            "md5": "430af00c0ed283f6281d9a83d389a1ef",
+        }
+        monkeypatch.setitem(CloudCoverDetection.image_meta, "test", test_image_meta)
+
+        test_target_meta = {
+            "filename": "ref_cloud_cover_detection_challenge_v1_test_labels.tar.gz",
+            "md5": "55813a5b2ca68473f508410a96419c6a",
+        }
+        monkeypatch.setitem(CloudCoverDetection.target_meta, "test", test_target_meta)
+
+        root = str(tmp_path)
+        split = "test"  # request.param
+        transforms = nn.Identity()  # type: ignore[no-untyped-call]
```

Comment:
```suggestion
        transforms = nn.Identity()
```
Latest version of PyTorch adds type hints for this